### ISSUE TITLE
No syntax highlighting on centos 7 by installing rpm

### ISSUE THREAD

**Issue by cg-cnu** [Microsoft/vscode#6012]
- VSCode Version:  1.0.0
- OS Version: Centos 7 KDE 3.10.0-123.el7.x86_64

Steps to Reproduce:

1. Install [vscode-x86_64.rpm](https://go.microsoft.com/fwlink/?LinkID=760867)
2. Open a project

No syntax highlighting.
Gave the following error.

![vscodebug](https://cloud.githubusercontent.com/assets/2767425/14935142/4df69a4a-0e95-11e6-83ee-1ad4544e9c86.png)
![vscodebug1](https://cloud.githubusercontent.com/assets/2767425/14935143/4dfc922e-0e95-11e6-977e-78d08b730015.png)

p.s. Previous installer worked fine on centos 7.

**Comment by bpasero** [comment]
Looks like the extension host crashes right on startup and thus no colouring applies.

@alexandrudima @jrieken fyi

**Comment by cg-cnu** [comment]
Last week I update the version to 1.1.0 and it worked fine without the above error.
I just installed the latest release 1.2.0 and got a new error message.
![vscodeerreor](https://cloud.githubusercontent.com/assets/2767425/15844735/53db799a-2c8a-11e6-99c0-07ffcbe239d2.png)
and syntax highlighting is not working.

**Comment by cg-cnu** [comment]
Ok, there is some confusion happening... 
I downloaded the latest rpm from this link... https://code.visualstudio.com/Updates#_downloads

I assume it will be version:  1.2.0
Once I install, in the help>about> Version is still showing as 1.1.1

**Comment by Tyriar** [comment]
@cg-cnu try installing via the command line, I find the GUI gets a little flaky sometimes:

```
sudo yum install <file>.rpm
```

**Comment by cg-cnu** [comment]
@Tyriar Yes, I was using cli all the time.

**Comment by Tyriar** [comment]
@cg-cnu what you linked are definitely the correct rpm packages at least in the region I'm being served in, I just downloaded both 32 and 64 bit and checked the package.json inside and it said 1.2.0.

**Comment by cg-cnu** [comment]
Hey... I just erased all the old installations and did a fresh install and that fixed the issue.
Sorry for the confusion, last time when I did yum install vscode it updated the rpm.
The yum installation is successful but not updating the actual files that could be the reason am seeing the old version in help menu and all the errors. I don't have the old rpm to install and try updating to the new one to test it.

This is fixed. Thanks for your time!
